omero/developers/Server/ScalingOmero.txt

```diff
@@ -55,15 +55,8 @@ The bottlenecks for throughput are:
 
 .. seealso::
     :doc:`/sysadmins/unix/server-postgresql`
-        Instructions about OMERO.server and PostgreSQL under UNIX & UNIX-like 
-        platforms.
-
-    :doc:`/sysadmins/windows/server-postgresql`
-        Instructions about OMERO.server and PostgreSQL under Windows  
-        platforms.
-
-
+        Instructions about OMERO.server and PostgreSQL under UNIX and
+        UNIX-like platforms.
 
     |OmeroGrid|
```

Comment:
There's a spacing issue with OmeroGrid here.  Does it need to be in bars, or could it be referenced directly?